branchtesting/src/main/java/branchexample/App.java

```diff
@@ -1,16 +1,23 @@
 package branchexample;
 
+import java.math.BigDecimal;
+
 public class App 
 {
+	public static String vulnerableString = new String();
+	
     public void aMethod( String[] args )
     {
-    	// TODO fix this
+    	double d = 1.1;
+
+    	BigDecimal bd1 = new BigDecimal(d); // Buggy
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove this "BigDecimal" constructor ([squid:S2129](https://sonarcloud.io/organizations/brian-cipollone-sonarsource-gith/rules#rule_key=squid%3AS2129))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=com.example.bcipollone%3Abranchtesting&pullRequest=12&issues=AWWImbQmXKENQn1SC_7O&open=AWWImbQmXKENQn1SC_7O)

[ISSUE_KEY]: AWWImbQmXKENQn1SC_7O